src/components/sidebar/linkView/LinkView.tsx

```diff
@@ -68,26 +117,32 @@ class LinkView extends React.Component<ILinkViewProps, ILinkViewState> {
                     <div className={s.flexInnerRow}>
                         <InputContainer
                             label='REITTITUNNUS'
-                            placeholder='1016'
+                            value={this.state.routePathLink!.routeId}
```

Comment:
This is a bit more of a generic point. This really shouldn't be changeable on it's own... Since this is part of a composite foreign key to routepath

Reply:
I can set them to disabled for now, but this will change